.Lib9c.Tests/Model/State/AgentStateTest.cs

```diff
@@ -0,0 +1,41 @@
+namespace Lib9c.Tests.Model.State
+{
+    using System.IO;
+    using System.Runtime.Serialization.Formatters.Binary;
+    using Libplanet;
+    using Libplanet.Crypto;
+    using Nekoyume.Model.State;
+    using Xunit;
+
+    public class AgentStateTest
+    {
+        [Fact]
+        public void Serialize()
+        {
+            var agentStateAddress = new PrivateKey().ToAddress();
+            var agentState = new AgentState(agentStateAddress);
+
+            var serialized = agentState.Serialize();
+            var deserialized = new AgentState((Bencodex.Types.Dictionary)serialized);
+
+            Assert.Equal(agentStateAddress, deserialized.address);
+        }
+
+        [Fact]
+        public void SerializeWithDotNetAPI()
+        {
+            var agentStateAddress = new PrivateKey().ToAddress();
+            var agentState = new AgentState(agentStateAddress);
+
+            var formatter = new BinaryFormatter();
+            using var ms = new MemoryStream();
+            formatter.Serialize(ms, agentState);
+            ms.Seek(0, SeekOrigin.Begin);
+            var deserialized = (AgentState)formatter.Deserialize(ms);
+
+            Assert.Equal(agentStateAddress, deserialized.address);
+
+            ms.Dispose();
```

Comment:
it's unnecessary because we're using [`using var` statement](https://docs.microsoft.com/ko-kr/dotnet/csharp/language-reference/keywords/using-statement).